### PR TITLE
Fixed assertion in one of the test and optimized retrieval of authentication header

### PR DESCRIPTION
**Pull request by chintans**
Fixed an assertion in AppServiceLoginHandlerTests/CreateToken_CreatesExpectedToken for validating timespan between issue and expiration of token. 

Optimized retrieval of authentication header from header with single call to TryGetValue.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@chintans__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

File: test/Microsoft.Azure.Mobile.Server.Login.Test/AppServiceLoginHandlerTests.cs [hunk 1]
```diff
@@ -3,6 +3,7 @@
 // ----------------------------------------------------------------------------
 
 using System;
+using System.Diagnostics;
```
Comment (brettsam) [inline test/Microsoft.Azure.Mobile.Server.Login.Test/AppServiceLoginHandlerTests.cs]: Please remove this using statement if it's not required.

**Comment by brettsam** [comment]
Hi @chintans. If you go ahead and remove that using Sysetm.Diagnostics, we can get this merged in. Thanks!

**Comment by chintans** [comment]
@brettsam I have removed using System.Diagnostics; as per your instruction. Can we merge it now?

**Comment by brettsam** [comment]
Looks good. Thanks for the contribution!

**Pull request merged**